gdsfactory/components/mmi.py

```diff
@@ -120,7 +120,7 @@ def mmi(
 
     x.add_bbox(c)
     c.auto_rename_ports()
-    c.flatten()
```

Comment:
I would suggest here to keep merge=Trye, otherwise VInstances with non45° do not work anymore